_documentation/user/flags/steamroller.md

```diff
@@ -5,4 +5,26 @@ type: Good
 summary: Destroys tanks you touch but you have to get really close.
 ---
 
-...
+Steamroller flag allows the player to kill tanks that get in a specified proximity of the holder.
```

Comment:
consider "configured" rather than "specified". The holder of the flag has no control of the radius; it is part of the map/world configuration.